AppImageBuilder.yml

```diff
@@ -0,0 +1,395 @@
+# appimage-builder recipe see https://appimage-builder.readthedocs.io for details
+version: 1
+
+script: |
+  mkdir -p build/AppDir build/AppDir/usr/share/icons/hicolor/scalable/apps
+  cp _kosmorro/assets/svg/kosmorro-icon.svg build/AppDir/usr/share/icons/hicolor/scalable/apps/kosmorro.svg
+  pip3 install --isolated --root="./build/AppDir" ./dist/kosmorro-*.whl
+
+AppDir:
+  path: build/AppDir
+  app_info:
+    id: space.kosmorro.cli
+    name: kosmorro
+    icon: kosmorro
+    version: latest
+    exec: usr/bin/python3
+    exec_args: $APPDIR/usr/local/bin/kosmorro $@
+
+  apt:
+    arch: amd64
+    allow_unauthenticated: true
+    sources:
+      - sourceline: deb http://archive.ubuntu.com/ubuntu/ focal main restricted
+      - sourceline: deb http://archive.ubuntu.com/ubuntu/ focal universe restricted
+    include:
+      - python3
+      - python3-dateutil
+      - texlive
+      - texlive-latex-extra
+
+  files:
+    include:
```

Comment:
It this necessary? They don't embed system libraries like this in the official examples: https://appimage-builder.readthedocs.io/en/latest/examples/pyqt.html

You should at least use a glob (it's supported; https://appimage-builder.readthedocs.io/en/latest/reference/version_1.html#files) instead of hardcoding this huge list of files. (Using a hardcoded list doesn't look solid: what if a library version changes or if a file is added)?

Reply:
Only a few system files were actually necessary, thanks :)